cmake/modules/RootConfiguration.cmake

```
@@ -854,6 +854,7 @@ configure_file(${CMAKE_SOURCE_DIR}/config/setxrd.sh ${CMAKE_RUNTIME_OUTPUT_DIREC
 configure_file(${CMAKE_SOURCE_DIR}/config/proofserv.in ${CMAKE_RUNTIME_OUTPUT_DIRECTORY}/proofserv @ONLY NEWLINE_STYLE UNIX)
 configure_file(${CMAKE_SOURCE_DIR}/config/roots.in ${CMAKE_RUNTIME_OUTPUT_DIRECTORY}/roots @ONLY NEWLINE_STYLE UNIX)
 configure_file(${CMAKE_SOURCE_DIR}/config/root-help.el.in root-help.el @ONLY NEWLINE_STYLE UNIX)
+configure_file(${CMAKE_SOURCE_DIR}/config/rootssh.sh ${CMAKE_RUNTIME_OUTPUT_DIRECTORY}/rootssh.sh @ONLY NEWLINE_STYLE UNIX)
```

```suggestion
configure_file(${CMAKE_SOURCE_DIR}/config/rootssh.sh ${CMAKE_RUNTIME_OUTPUT_DIRECTORY}/rootssh @ONLY NEWLINE_STYLE UNIX)
```
We generally call "things" without ".py" or ".sh" extension - that's an implementation detail :-)

---

Maybe rename `rootssh.sh` into `rootssh` already in sources?
Name of script also appears in `INSTALL` instructions.

And in general script can be used without ROOT on local machine.

---

I will change this name in next PR.